e2e-tests/cypress/e2e/journeys/insurance/start.spec.js

```diff
@@ -69,6 +69,36 @@ context('Insurance Eligibility - start page', () => {
 
       cy.checkText(insurance.startPage.body4(), CONTENT_STRINGS.BODY_4);
     });
+
+    it('renders a "sign-in" body text', () => {
+      const { SIGN_IN } = CONTENT_STRINGS;
+
+      cy.checkText(insurance.startPage.signIn.youCan(), SIGN_IN.YOU_CAN);
+
+      cy.checkLink(insurance.startPage.signIn.link(), SIGN_IN.LINK.HREF, SIGN_IN.LINK.TEXT);
+
+      cy.checkText(insurance.startPage.signIn.toContinueApplication(), SIGN_IN.TO_CONTINUE_APPLICATION);
+    });
+
+    it('renders a "find out more" body text', () => {
+      const { FIND_OUT_MORE } = CONTENT_STRINGS;
+
+      cy.checkText(insurance.startPage.findOutMore.youCan(), FIND_OUT_MORE.YOU_CAN);
+
+      cy.checkLink(insurance.startPage.findOutMore.link(), FIND_OUT_MORE.LINK.HREF, FIND_OUT_MORE.LINK.TEXT);
+
+      cy.checkText(insurance.startPage.findOutMore.toFindOutMore(), FIND_OUT_MORE.TO_FIND_OUT_MORE);
+    });
+
+    it('renders a "get a quote" body text', () => {
+      const { QUOTE } = CONTENT_STRINGS;
+
+      cy.checkText(insurance.startPage.getAQuote.youCan(), QUOTE.YOU_CAN);
+
+      cy.checkLink(insurance.startPage.getAQuote.link(), QUOTE.LINK.HREF, QUOTE.LINK.TEXT);
+
+      cy.checkText(insurance.startPage.getAQuote.ifYouNeed(), QUOTE.IF_YOU_NEED);
+    });
```

Comment:
Not a blocker, could destructure the sections from `insurance.startPage`

Reply:
will do in next PR